pdr-admin/src/env.js

```diff
@@ -9,7 +9,7 @@
   // Environment name
   window.__env.ENVIRONMENT = "local"; // local | dev | test | prod
 
-  window.__env.API_LOCATION = "http://localhost:3000";
+  window.__env.API_LOCATION = "https://zloys5cfvf.execute-api.ca-central-1.amazonaws.com";//"http://localhost:3000";
```

Comment:
I don't think we should change this.

Reply:
fixed in [c37476f](https://github.com/bcgov/parks-data-register/pull/192/commits/c37476fa5ca9a4dc9435b661c605838b66b9b871)